backend/src/@types/fastify.d.ts

```diff
@@ -19,6 +19,8 @@ import { TGatewayServiceFactory } from "@app/ee/services/gateway/gateway-service
 import { TGatewayV2ServiceFactory } from "@app/ee/services/gateway-v2/gateway-v2-service";
```

Comment:
IGNORE PLACEMENT OF COMMENT

I think the backend is missing `identityGroups` for the plan? I can't seem to add any groups
<img width="1512" height="857" alt="image" src="https://github.com/user-attachments/assets/d30021d1-9e7d-49d4-b6f3-c45fa1fe4d39" />